app/controllers/application_controller.rb

```diff
@@ -88,6 +89,22 @@ def sublayout
 
   protected
 
+  def check_browser
+    request_format = request.format
+    return true if request_format.xml? || request_format.json?
+
+    if current_account&.provider_can_use?(:modern_browser_check) && browser_not_modern?
```

Comment:
I think we should keep the number of switches to a minimum as it only leads to edge cases and complexity. Maybe just enable for all? What do you think @hallelujah @guicassolato?

Reply:
Agree, we can enable for all yes.